ReactApps/reacttest/src/wrapper.jsx

```diff
@@ -0,0 +1,59 @@
+
+import React, {Component} from "react";
+import ItemCard from "./card";
+
+class Wrapper extends Component {
+
+    constructor(props) {
+        super(props);
+        this.id = 0;
+        this.state = {
+            data: [
```

Comment:
Do we need this initial state if its empty ? Please remove comments.

Reply:
yes, it's need to render empty catalog